doc/unit_test.rst

```diff
@@ -67,6 +67,16 @@ Unit Tests for mlx.traceability
 
 .. item:: UTEST_ITEM_DIRECTIVE-MAKE_INTERNAL_ITEM_REF_SHOW_CAPTION
 
+.. item:: UTEST_REP_TRACEABLE_ITEM-SET_CAPTION
+    :passes: UTEST_TRACEABLE_ITEM-SET_CAPTION
+    :result: pass
+
+.. item:: UTEST_REP_ITEM_DIRECTIVE-MAKE_INTERNAL_ITEM_REF_SHOW_CAPTION
+    :skipped: UTEST_ITEM_DIRECTIVE-MAKE_INTERNAL_ITEM_REF_SHOW_CAPTION
+..
+    due to the lack of an attribute (value) 'result' above, RQT-CAPTION will be labeled as 'executed' instead of 'skipped'
+    in an item-piechart with ``::result: ERROR, fail, pass, skip``
```

Comment:
This is a bit deceiving because it contains `skipped` attribute. Is there a use case where item would truly be shown as skipped, or is this the case? Maybe name this item `_EXECUTED` and anotherone `_SKIPPED` to show this.

Reply:
It does not contain `skipped` attribute, that's the point. It only contains a `skipped` relationship/targettype. 

If the item-matrix uses targettype to label the slices, the corresponding source item would likely show up in the `skipped` slice.

I'm testing a pie chart (see code block below) that uses both the `:targettype:` option and the `:<<attribute>>:` option (`:result:`), of which the latter has precedence. The TEST_REP item shows up in the matrix, but due to the lack of a `:result: skip` in its item definition, it is labeled as `has report`.

```
.. item-piechart:: Test coverage chart with test results, based on the :result: attribute
    :id_set: RQT [IU]TEST [IU]TEST_REP
    :sourcetype: validated_by
    :targettype: failed_by passed_by skipped_by
    :label_set: uncovered, covered, has report
    :result: ERROR, fail, pass, skip
    :colors: orange c b darkred #FF0000 g pink
    :stats:
    :matrix:
    ```
    
    I just now committed 13f98861814ef120de77f913960e9b39ea31577d